lib/ui/onboarding.dart

```diff
@@ -32,102 +34,111 @@ class _OnboardingScreenState extends State<OnboardingScreen> {
-                                style: TextStyle(
-                                    decoration: TextDecoration.lineThrough)),
-                            TextSpan(
-                                text: 'roundExplanationEnd'.tr)
-                          ])),
-                  Stack(clipBehavior: Clip.none, children: [
-                    SizedBox(
-                        width: mediaWidth,
-                        child: SvgPicture.asset('assets/images/carWithLulo.svg',
-                            width: size.width)),
-                    Positioned(
-                        bottom: -15.0,
-                        left: mediaWidth / 2 - mediaWidth / 6,
-                        child: SizedBox(
-                            width: mediaWidth / 3,
-                            height: 48,
-                            child: ElevatedButton(
-                                onPressed: () {
-                                  Get.toNamed(UserNameScreen.route);
-                                },
-                                child: Text(
-                                  'continue'.tr,
-                                ))))
-                  ])),
-            ],
-          ),
-          Positioned(
-              left: 0.0,
-              right: 0.0,
-              bottom:  80.0,
-              child: CirclePageIndicator(
-                size: 10.0,
-                selectedSize: 12.0,
-                itemCount: 3,
-                currentPageNotifier: _currentPageNotifier,
-              )
-          ),
-        ]
-    );
+    return PageView(
+      scrollDirection: Axis.horizontal,
+      controller: controller,
+      onPageChanged: (int index) {
+        _currentPageNotifier.value = index;
+        if(index == 3){
+          Get.toNamed(UserNameScreen.route);
+        }
+      },
+      children: [
+        Stack(
+          children:[
+            pageBuilder(
+              Text('haveYouHeard...'.tr,
+                style: HyhTextStyle.body16Height15),
+              RichText(
+                text: TextSpan(
+                    text: 'gameExplanation'.tr,
+                    style: HyhTextStyle.body16Height15)),
+              Hero(
+                tag: 'logo',
+                child: Container(
+                    margin: const EdgeInsets.symmetric(vertical: 0, horizontal: 47),
+                    alignment: Alignment.topRight,
+                    height: logoHeight,
+                    width: size.width,
+                    child: SvgPicture.asset('assets/images/logo.svg',
+                        width: mediaWidth / 3))),
+            ),
+            Positioned(
+                left: 0.0,
+                right: 0.0,
+                bottom:  80.0,
+                child: CirclePageIndicator(
+                  size: 10.0,
+                  selectedSize: 12.0,
+                  itemCount: 3,
+                  currentPageNotifier: _currentPageNotifier,
+                )),
```

Comment:
The CirclePageIndicator is being swiped together with the page, it is supposed to stay still.